src/components/services-view/services-view.controller.ts

```diff
@@ -180,4 +181,35 @@ export class ServicesViewController implements angular.IController {
   private updateActiveCardStyles() {
     this.$timeout(this.resizeExpansion, 50);
   }
+
+  private setRecentlyViewed() {
```

Comment:
It's probably worth having a recently viewed service for managing this. Then we can add items from other places (like search) and the projects summary can get items without having to know that they're kept in localStorage.